types/index.d.ts

```diff
@@ -11,6 +11,7 @@ export type SonicBoomOpts = {
     dest?: string | number
     maxLength?: number
     minLength?: number
+    maxWrite?: number
```

Comment:
We actually test our types with tsd! Could you add an assertion for this?

Reply:
@mcollina please correct me if I am wrong - by checking out how **tsd** works and checking out some other pino repositories I guess I only need to make change like this https://github.com/pinojs/sonic-boom/pull/142/commits/42799c6ce275afd76f447a7d57b0ebf075618605


Let me know if my understanding is bad, and thanks for sharing with me **tsd**

Reply:
that's ok!